src/extension/mapml/src/test/java/org/geoserver/mapml/MapMLWMTSProxyTest.java

```diff
@@ -0,0 +1,160 @@
+/* (c) 2024 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.mapml;
+
+import static org.geowebcache.grid.GridSubsetFactory.createGridSubSet;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Map;
+import org.geoserver.catalog.Catalog;
+import org.geoserver.catalog.LayerInfo;
+import org.geoserver.catalog.ResourceInfo;
+import org.geoserver.catalog.WMTSLayerInfo;
+import org.geoserver.catalog.WMTSStoreInfo;
+import org.geoserver.data.test.SystemTestData;
+import org.geoserver.gwc.GWC;
+import org.geoserver.gwc.config.GWCConfig;
+import org.geoserver.gwc.layer.GeoServerTileLayer;
+import org.geoserver.mapml.gwc.gridset.MapMLGridsets;
+import org.geowebcache.grid.GridSubset;
+import org.geowebcache.mime.TextMime;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class MapMLWMTSProxyTest extends MapMLBaseProxyTest {
+
+    protected static final String BASE_WMTS_REQUEST =
+            "wms?LAYERS=cascadedLayer"
+                    + "&STYLES=&FORMAT="
+                    + MapMLConstants.MAPML_MIME_TYPE
+                    + "&SERVICE=WMS&VERSION=1.1.0"
+                    + "&REQUEST=GetMap"
+                    + "&SRS=MapML:OSMTILE"
+                    + "&BBOX=-1.3885038382960921E7,2870337.130793682,-7455049.489182421,6338174.0557576185"
+                    + "&WIDTH=768"
+                    + "&HEIGHT=414"
+                    + "&format_options="
+                    + MapMLConstants.MAPML_WMS_MIME_TYPE_OPTION
+                    + ":image/png";
+
+    @Override
+    protected void registerNamespaces(Map<String, String> namespaces) {
```

Comment:
Wondering if all the namespaces can just be declared in the base class... or are there conflicts?